swarm/network/kademlia.go

```diff
@@ -224,6 +225,7 @@ func (k *Kademlia) SuggestPeer() (a OverlayAddr, o int, want bool) {
 		if po < depth {
 			return false
 		}
+		//log.Warn("val suggest", "val", fmt.Sprintf("%p", val))
```

Comment:
cleanup commented code (not mentioning again from now on; @zelig has been quite insisting on this on my last PR)